tests/Fixtures/e2e/Custom_tmp_dir/tests/SourceClassTest.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Namespace_\Test;
+
+use Namespace_\SourceClass;
+
+class SourceClassTest extends \PHPUnit\Framework\TestCase
```

Comment:
Not a big deal but I find better to be consistent and import this class with a use statement?